playwright-e2e/tests/singular/enrollment/adult-dependent-enrollment-visual.spec.ts

```diff
@@ -0,0 +1,101 @@
+import * as auth from 'tests/lib/auth-singular';
+import MyDashboardPage from '../dashboard/my-dashboard-page';
+import EnrollMyAdultDependentPage from './enroll-my-adult-dependent-page';
+import PreScreeningPage from './pre-screening-page';
+import * as nav from '../lib/nav';
+import { WHO } from 'data/constants';
+import { makeEmailAlias } from 'utils/string-utils';
+
+
+async function signUp(page: Page) {
+  await nav.signMeUp(page);
+
+  const preScreeningPage = new PreScreeningPage(page);
+  await preScreeningPage.enterInformationAboutYourself();
+
+  // Enter email alias and new password in Login popup
+  await fillEmailPassword(page, {
+    email: makeEmailAlias(process.env.singularUserEmail as string),
+    password: process.env.singularUserPassword,
+    waitForNavigation: true
+  });
+}
+
+
+test.describe('Adult Dependent visual tests', () => {
+  test.beforeEach(async ({ page }) => {
+    await nav.goToPath(page, '/password');
+    await auth.fillSitePassword(page);
+    await new HomePage(page).waitForReady();
+    await signUp(page);
+
+    const myDashboardPage = new MyDashboardPage(page);
+    await myDashboardPage.enrollMyAdultDependent();
+  });
+
+  // whoHasVentricleHeartDefect validation: Select `someone else` should trigger an error message
+  test('select a "Someone else" option for adult dependent @visual @enrollment @singular', async ({ page }) => {
+    // On "Enroll my adult dependent" page
+    const enrollMyAdultDependentPage = new EnrollMyAdultDependentPage(page);
+    const whoHasVentricleHeartDefect = enrollMyAdultDependentPage.whoHasVentricleHeartDefect();
+    // No error before select who has ventricle heart defect
+    await expect(whoHasVentricleHeartDefect.errorMessage()).toBeHidden();
+
+    await whoHasVentricleHeartDefect.check(WHO.SomeoneElse);
+    await expect(whoHasVentricleHeartDefect.errorMessage()).toContainText(
+      'Only an individual with single ventricle heart defect or immediate family members of an individual with single ventricle heart defect can be enrolled.'
+    );
+
+    let screenshotError = await whoHasVentricleHeartDefect.toLocator().screenshot();
+    expect(screenshotError).toMatchSnapshot('whoHasVentricleHeartDefect-error-message.png');
+
+    // Clear the error message by select `The dependant being enrolled`
+    await whoHasVentricleHeartDefect.uncheck(WHO.SomeoneElse);
+    await whoHasVentricleHeartDefect.check(WHO.TheDependantBeingEnrolled);
+    await page.waitForResponse((resp) => resp.url().includes('/answers') && resp.status() === 200);
+    let screenshotSuccess = await whoHasVentricleHeartDefect.toLocator().screenshot();
+    expect(screenshotSuccess).toMatchSnapshot('whoHasVentricleHeartDefect-no-errors.png');
+  });
+
+  // Age validation: should be 18 y.o. or more
+  test('should validate adult dependent age @visual @enrollment @singular', async ({ page }) => {
+    const enrollMyAdultDependentPage = new EnrollMyAdultDependentPage(page);
```

Comment:
It seems that we can move this line as part of the beforeEach block (it is presenting on every test).

Reply:
Fixed.